tc/graphene/python_worker/docker-compose.yaml

```diff
@@ -19,8 +19,8 @@ services:
     container_name: process-work-order
     image: avalon-python-worker-dev
     build:
-      context: .
-      dockerfile: ./Dockerfile
+      context: ../../..
+      dockerfile: ./tc/graphene/python_worker/Dockerfile
```

Comment:
Why we had to change this context in this PR ?

Reply:
This was done to copy crypto files from common/crypto_utils/avalon_crypto_utils to python worker image

Reply:
Please add a comment in this file.

Reply:
Done